doc/book/box/atomic.rst

```diff
@@ -244,3 +244,27 @@ via telnet, via one of the connectors, or via the
 
 After a fiber has yielded and then has regained control, it immediately issues
 :ref:`testcancel <fiber-testcancel>`.
+
+.. atomic-transactional-manager
+
+--------------------------------------------------------------------------------
+Transactional manager
+--------------------------------------------------------------------------------
+
+Since version 2.6.1, Tarantool has another option for transaction behaviour that
+allows yielding inside a memtx transaction. This is controled by
+the *transactional manager*.
+
+The transactional manager is designed for isolation of concurrent transactions.
+It consists of two parts:
+
+* *MVCC engine*
+* *conflict manager*.
+
+The MVCC engine provides personal read views for transactions if necessary.
+The conflict manager tracks transactions' changes and determines their correctness
+in serialization order. Of course, once yielded, a transaction could interfere
+with other transactions and could be aborted due to conflict.
```

Comment:
Another thing that worth to be mentioned is that the transaction manager provides non classic snapshot isolation level. That means that a transaction can get a consistent snapshot of DB (that's common) but that snapshot is not necessary bound with the moment of the beginning of the transaction (that's not common). Conflict manager makes decisions of whether and when each transaction gets which snapshot. That allows to avoid some conflicts comparing with classical snapshot isolation approach.

Reply:
Added this info